### PR TITLE
Refactor ML.NET ONNX Object Detection Console Application Sample

### PR DESCRIPTION
**Pull request by luisquintanilla**
- Refactor `ParseOutputs` to be more readable instead of aligned with academic paper.
- Add functionality to draw bounding boxes on image.

### CODE REVIEW

File: samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/ObjectDetectionConsoleApp/DataStructures/ImageNetData.cs [hunk 1]
```diff
@@ -1,11 +1,11 @@
-﻿using Microsoft.ML.Data;
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
+﻿#region ImageNetDataUsings
```
Comment (prathyusha12345) [inline samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/ObjectDetectionConsoleApp/DataStructures/ImageNetData.cs]: Please remove these #region ....we need to keep the code consistent with other samples

File: samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/ObjectDetectionConsoleApp/DataStructures/ImageNetData.cs [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+using Microsoft.ML.Data;
+#endregion
 
 namespace ObjectDetection
 {
+
+    #region ImageNetDataClass
```
Comment (prathyusha12345) [inline samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/ObjectDetectionConsoleApp/DataStructures/ImageNetData.cs]: remove #region

File: samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/ObjectDetectionConsoleApp/DataStructures/ImageNetData.cs [hunk 3]
```diff
@@ -14,10 +14,14 @@ public class ImageNetData
         [LoadColumn(1)]
         public string Label;
     }
+    #endregion
 
+    #region ImageNetDataProbabilityClass
```
Comment (prathyusha12345) [inline samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/ObjectDetectionConsoleApp/DataStructures/ImageNetData.cs]: remove #region

File: samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/ObjectDetectionConsoleApp/OnnxModelScorer.cs [hunk 4]
```diff
@@ -1,28 +1,34 @@
-﻿using System;
+﻿#region OnnxModelScorerUsings
```
Comment (prathyusha12345) [inline samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/ObjectDetectionConsoleApp/OnnxModelScorer.cs]: remove #region

File: samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/ObjectDetectionConsoleApp/DataStructures/ImageNetData.cs (hunk 1 above)
Comment (luisquintanilla) [reply samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/ObjectDetectionConsoleApp/DataStructures/ImageNetData.cs]: #resolved

File: samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/ObjectDetectionConsoleApp/DataStructures/ImageNetData.cs (hunk 3 above)
Comment (luisquintanilla) [reply samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/ObjectDetectionConsoleApp/DataStructures/ImageNetData.cs]: #resolved

File: samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/ObjectDetectionConsoleApp/OnnxModelScorer.cs (hunk 4 above)
Comment (luisquintanilla) [reply samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/ObjectDetectionConsoleApp/OnnxModelScorer.cs]: #resolved

File: samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/ObjectDetectionConsoleApp/DataStructures/ImageNetData.cs (hunk 2 above)
Comment (luisquintanilla) [reply samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/ObjectDetectionConsoleApp/DataStructures/ImageNetData.cs]: #resolved

**Comment by prathyusha12345** [comment]
As discussed with @luisquintanilla Luis , currently the changes in the PR are as below.

The code in YoloMLParser.cs class is extracted into  more meaningful methods .
Added code to draw bounding boxes on the images and the images are stored in output folder.

The changes look good to me. I am merging the PR. Thanks

**Pull request merged**